### PR TITLE
[FIX] l10n_it_edi: Codes in XML template shouldn't be translated

### PR DESCRIPTION
**Pull request by lordkrandel**
Codes like "Exch.Rate" in the Italian EDI XML template for invoices were translated.
They shouldn't be, as they have pretty short char limit and it's risky to people change that. The XML users are either domestic or the Italian Tax Agency itself, so no point in translating "Divisa" into "Currency" anyway.

Link: https://www.odoo.com/web#model=project.task&id=3627379 
opw-3627379

### CODE REVIEW

**Comment by lordkrandel** [comment]
@jco-odoo  - Yet I don't understand why these tags in the template are in English and get translated. They shouldn't be language dependent.

File: addons/l10n_it_edi/data/invoice_it_template.xml [hunk 1]
```diff
@@ -6,11 +6,11 @@
                 <DettaglioLinee>
                     <NumeroLinea t-esc="line_dict['line_number']"/>
                     <CodiceArticolo t-if="line.product_id.barcode">
-                        <CodiceTipo>EAN</CodiceTipo>
+                        <CodiceTipo t-translation="off">EAN</CodiceTipo>
                         <CodiceValore t-esc="format_alphanumeric(line.product_id.barcode)[:35]"/>
                     </CodiceArticolo>
                     <CodiceArticolo t-elif="line.product_id.default_code">
-                        <CodiceTipo>INTERNAL</CodiceTipo>
+                        <CodiceTipo t-translation="off">INTERNAL</CodiceTipo>
```
Comment (rugo-odoo) [inline addons/l10n_it_edi/data/invoice_it_template.xml:13]:
```suggestion
                        <CodiceTipo t-translation="off">INTERNO</CodiceTipo>
```

**Review by rugo-odoo — CHANGES_REQUESTED** [review]
Good job ! Just a little comment.

File: addons/l10n_it_edi/data/invoice_it_template.xml (hunk 1 above)
Comment (lordkrandel) [reply addons/l10n_it_edi/data/invoice_it_template.xml:13]: No, you're right, I don't think he knows it's just a formal code to indicate the meaning of the field. Can you ask hkm?

Comment (rugo-odoo) [reply addons/l10n_it_edi/data/invoice_it_template.xml:13]: I don't understand. Asking what to who ? It's your task, you should be doing the asking 😃 (I don't have as much understanding/knowledge about it as you and it is more efficient if you ask him/her directly.)

Comment (lordkrandel) [reply addons/l10n_it_edi/data/invoice_it_template.xml:13]: Yeah, but I'm still away today, and can't ping him from the phone app. Just asked as a favour to speed up things but ok nvm. :D Will do tomorrow.

**Comment by tsb-odoo** [comment]
LGTM @rugo-odoo

**Comment by rugo-odoo** [comment]
(cc @william-andre)

File: addons/l10n_it_edi/data/invoice_it_template.xml (hunk 1 above)
Comment (william-andre) [reply addons/l10n_it_edi/data/invoice_it_template.xml:13]: Just to be sure, is this resolved?

Comment (lordkrandel) [reply addons/l10n_it_edi/data/invoice_it_template.xml:13]: What do you mean by "resolved"?
- Resolved as in "retrieved by the translation framework" ? Yes, see the translation I removed here: https://github.com/odoo/odoo/pull/153111/files#diff-8f09903121e8b975eada67a02a995dd1e2cd5fa68f2755147dc7804d1c787e4bL368
- Resolved meaning "solved" - yes, here TSB approves: https://github.com/odoo/odoo/pull/153111#issuecomment-1967266813
- Other meaning that I'm missing?

Comment (lordkrandel) [reply addons/l10n_it_edi/data/invoice_it_template.xml:13]: Yes.

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**